dev/build/windows/makecoq_mingw.sh

```diff
@@ -1023,6 +1024,16 @@ function make_num {
   fi
 }
 
+function make_zarith {
+  make_ocaml
+  if build_prep https://github.com/ocaml/Zarith/archive release-1.9.1 tar.gz 1 zarith-1.9.1; then
+    log1 ./configure
```

Comment:
@ejgallego : this does not work. log1 assumes that the first argument is at the same time a command and a valid *piece* of a log file name. If the command contains e.g. /, you should use logn, where the first argument is an arbitrary log name and the rest is the command. Typical usage is `logn configure ./configure`.

This is easy to see in the log - just scroll above the artefact upload (which looks very regular).

Reply:
Thanks, I was indeed confused by this, if I tried `log1 configure` it said "configure program not found".

Reply:
> Thanks, I was indeed confused by this, if I tried `log1 configure` it said "configure program not found".

yes, as you know this is usual posix shell behaviour.

In all non trivial cases use logn (for log name). log1 and log2 are just abbreviations which reuse the first (and second) word of the command as log file name postfix(es). logn is the generic variant.